### ISSUE TITLE
[dev.icinga.com #11871] Colors used in the timeline are not accessible

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2442]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/11871

**Created by elippmann on 2016-06-01 11:05:11 +00:00**

Assignee: _(none)_
Status: _Resolved_  (closed on _2016-06-01 12:45:03 +00:00_)
Target Version: _2.3.4_
Last Update: _2016-06-01 12:45:03 +00:00 (in Redmine)_

---

**Changesets**

_2016-06-01 12:40:55 +00:00_ by _elippmann_ ba5fe61fa921058d2e05f89c214d57036717555a

    monitoring: Make timeline colors accessible
    
    fixes #11871
